### PR TITLE
Karl/homepage content

### PR DESCRIPTION
**Pull request by Karl-J-Bruder**
Decided to try working on some of the homepage content. This PR only has the event signup steps right now, but I plan to add the other ones later.

### CODE REVIEW

File: src/components/organisms/SignupPanel/SignupPanel.story.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import SignupPanel from "./index"
+
+const story = storiesOf("Organisms | SignupPanel", module)
+    .addDecorator(withInfo)
+    .addDecorator(withKnobs)
+// Signup instructions appearing on homepage
+story.add("Component", () => (
+    <SignupPanel
+        blockTitle={text("Block title", "Your adventure starts here!")}
+        buttonColor={"redWhite"}
```
Comment (Alfrex92) [inline src/components/organisms/SignupPanel/SignupPanel.story.tsx]: You can do something like this:

`{color("Color", teal)}`

File: src/components/organisms/SignupPanel/SignupPanel.story.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { storiesOf } from "@storybook/react"
+import { withKnobs, text } from "@storybook/addon-knobs"
```
Comment (Alfrex92) [inline src/components/organisms/SignupPanel/SignupPanel.story.tsx]: you need to import `color`

File: src/components/organisms/SignupPanel/index.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import data from "../../../../config/translations/en.json"
```
Comment (Alfrex92) [inline src/components/organisms/SignupPanel/index.tsx:2]: Why you're importing data directly to the component? What if the data is in Japanese?

Remember **everything comes from props**

Check the Members component that I did

File: src/components/atoms/HPCBlockTitle/HPCBlockTitle.story.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import { storiesOf } from "@storybook/react"
+import { withKnobs, text } from "@storybook/addon-knobs"
+import { withInfo } from "@storybook/addon-info"
+
+import HPCBlockTitle from "./index"
```
Comment (Alfrex92) [inline src/components/atoms/HPCBlockTitle/HPCBlockTitle.story.tsx:6]: what does `HPCB` stands for?

File: src/components/atoms/HPCBlockTitle/index.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+import { StyleHPCBlockTitleWrapper, StyleHPCBlockTitle } from "./styles"
+
+interface HPCBlockTitleInterface {
+    title: string,
+}
+
+const HPCBlockTitle: React.FC<HPCBlockTitleInterface> = (
+    props: HPCBlockTitleInterface
+) => {
+    console.log(props.title)
```
Comment (Alfrex92) [inline src/components/atoms/HPCBlockTitle/index.tsx:11]: don't commit console logs to develop branch

**Review by Alfrex92 — COMMENTED** [review]
I left some comments

File: src/components/organisms/SignupPanel/SignupPanel.story.tsx (hunk 1 above)
Comment (Karl-J-Bruder) [reply src/components/organisms/SignupPanel/SignupPanel.story.tsx]: Ok, I reworked it to accept hex values for the colors (as strings), which seems to work:

    buttonColor={color("Button color", "#f92b00")}
    hasBorder={boolean("Border?", false)} <--Applies border of same color as text if true, transparent if false
    textColor={color("Button color", "#fff")}

File: src/components/organisms/SignupPanel/SignupPanel.story.tsx (hunk 2 above)
Comment (Karl-J-Bruder) [reply src/components/organisms/SignupPanel/SignupPanel.story.tsx]: OK. Added color and boolean (see above comment)

**Comment by Alfrex92** [comment]
Nice, I think that you now merge this PR. It's approved.

**Review by Alfrex92 — APPROVED** [review]
LGTM. you can now merge this PR.

File: src/components/organisms/SignupPanel/index.tsx (hunk 3 above)
Comment (Karl-J-Bruder) [reply src/components/organisms/SignupPanel/index.tsx:2]: Yeah, I keep forgetting that. I fixed it though, so it works on Storybook.

File: src/components/atoms/HPCBlockTitle/HPCBlockTitle.story.tsx (hunk 4 above)
Comment (Karl-J-Bruder) [reply src/components/atoms/HPCBlockTitle/HPCBlockTitle.story.tsx:6]: Home Page Content Block
(I'll remember to leave a comment in the code if I use abbreviations)

File: src/components/atoms/HPCBlockTitle/index.tsx (hunk 5 above)
Comment (Karl-J-Bruder) [reply src/components/atoms/HPCBlockTitle/index.tsx:11]: Oops! Good catch. I deleted all the console logs I'd left in the files.

**Pull request merged**